tests/test_on_data/test_format_converters/test_spikeglx_converter.py

```diff
@@ -91,10 +89,66 @@ class TestConverter(NWBConverter):
         nwbfile_path = self.tmpdir / "test_spikeglx_converter_in_nwbconverter.nwb"
         converter.run_conversion(nwbfile_path=nwbfile_path)
 
-        self.assertNWBFileStructure(nwbfile_path=nwbfile_path)
+        expected_session_start_time = datetime(2020, 11, 3, 10, 35, 10).astimezone()
+        self.assertNWBFileStructure(nwbfile_path=nwbfile_path, expected_session_start_time=expected_session_start_time)
+
+
+class TestMultiProbeSpikeGLXConverter(TestCase):
+    maxDiff = None
+
+    def setUp(self):
+        self.tmpdir = Path(mkdtemp())
+
+    def tearDown(self):
+        rmtree(self.tmpdir)
+
+    def assertNWBFileStructure(self, nwbfile_path: FilePath, expected_session_start_time: datetime):
+        with NWBHDF5IO(path=nwbfile_path) as io:
+            nwbfile = io.read()
+
+        assert nwbfile.session_start_time == expected_session_start_time
+
+        assert "ElectricalSeriesAPIMEC00" in nwbfile.acquisition
+        assert "ElectricalSeriesAPIMEC01" in nwbfile.acquisition
+        assert "ElectricalSeriesAPIMEC10" in nwbfile.acquisition
+        assert "ElectricalSeriesAPIMEC11" in nwbfile.acquisition
+
+        assert "ElectricalSeriesLFIMEC00" in nwbfile.acquisition
+        assert "ElectricalSeriesLFIMEC01" in nwbfile.acquisition
+        assert "ElectricalSeriesLFIMEC10" in nwbfile.acquisition
+        assert "ElectricalSeriesLFIMEC11" in nwbfile.acquisition
+
+        assert "Neuropixel-Imec" in nwbfile.devices
+
+        assert "s0" in nwbfile.electrode_groups
```

Comment:
This is only producing one group, right? It should produce one group per probe I think. And in fact, the electrodes table is not having an electrode for every channel here:

```python
from neuroconv.converters import SpikeGLXConverterPipe

SPIKEGLX_PATH = ECEPHY_DATA_PATH / "spikeglx"
folder_path=SPIKEGLX_PATH / "multi_trigger_multi_gate" / "SpikeGLX" / "5-19-2022-CI0" / "5-19-2022-CI0_g0"

converter = SpikeGLXConverterPipe(folder_path=folder_path)
nwbfile = converter.create_nwbfile()
nwbfile.electrodes.to_dataframe()
```

I think this is related to this bug here:

https://github.com/catalystneuro/neuroconv/issues/958